.merlin

```diff
@@ -20,6 +20,7 @@ B _build/lib/bap_plugins
 B _build/lib/text_tags
 B _build/lib/regular
 B _build/lib/graphlib
+B _build/lib/microx
```

Comment:
Hmm.. it has this line https://github.com/BinaryAnalysisPlatform/bap/blob/master/plugins/propagate_taint/.merlin#L3 but for some reason didn't work. Maybe I didn't reinstall with the microx package.